samples/Accelerometer.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) 2016, Intel Corporation.
+
+// Test code to use the Accelerometer (subclass of Generic Sensor) API
+// to communicate with the BMI160 inertia sensor on the Arduino 101
+// and obtaining information about acceleration applied to the X, Y and Z axis
+console.log("Accelerometer test...");
+
+
+var sensor = new Accelerometer({
+    includeGravity: false, // true is not supported, will throw error
+    frequency: 50
+});
+
+sensor.onchange = function(event) {
```

Comment:
@jimmy-huang : Some changes were made for the [Sensor Reading](https://w3c.github.io/sensors/#sensor-reading)
Now
```js
accleromterSensor.onchange = function () {
console.log("acceleration (m/s^2): " +
        " x=" + accleromterSensor.reading.x +
        " y=" + accleromterSensor.reading.y +
        " z=" + accleromterSensor.reading.z);
}
```
just a heads up, you can make this changes in subsequent PR also, upto you.